frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1221,6 +1235,99 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                     ]
                 }
 
+                if (productTab === ProductTab.MARKETING) {
```

Comment:
Not related to your PR, but now that we're adding more tabs, we should consider refactoring this main kea logic into more specific ones. There is some shared logic for sure, but 80% is the query payload creation.